frontend/packages/shipwright-plugin/console-extensions.json

```diff
@@ -98,87 +98,179 @@
+      "disallowed": ["SHIPWRIGHT_BUILDRUN", "SHIPWRIGHT_BUILD", "SHIPWRIGHT_BUILD_V1ALPHA1"]
     }
   },
   {
-    "type": "console.navigation/resource-ns",
+    "type": "console.navigation/href",
     "properties": {
+      "id": "shipwright-tablistpage-5",
+      "section": "builds",
       "perspective": "admin",
+      "prefixNamespaced": true,
+      "href": "/shipwright.io",
+      "name": "%shipwright-plugin~Shipwright%"
+    },
+    "flags": {
+      "required": ["SHIPWRIGHT_BUILDSTRATEGY_V1ALPHA1"],
+      "disallowed": ["SHIPWRIGHT_BUILDRUN_V1ALPHA1", "SHIPWRIGHT_BUILDRUN", "SHIPWRIGHT_BUILD", "SHIPWRIGHT_BUILD_V1ALPHA1"]
+    }
+  },
+  {
+    "type": "console.navigation/href",
+    "properties": {
+      "id": "shipwright-tablistpage-6",
       "section": "builds",
-      "id": "shipwright-builds",
-      "name": "%shipwright-plugin~Shipwright Builds%",
-      "model": {
-        "group": "shipwright.io",
-        "version": "v1beta1",
-        "kind": "Build"
-      }
+      "perspective": "admin",
+      "prefixNamespaced": true,
+      "href": "/shipwright.io",
+      "name": "%shipwright-plugin~Shipwright%"
     },
     "flags": {
-      "required": ["SHIPWRIGHT_BUILD"]
+      "required": ["SHIPWRIGHT_BUILDSTRATEGY"],
+      "disallowed": ["SHIPWRIGHT_BUILDSTRATEGY_V1ALPHA1", "SHIPWRIGHT_BUILDRUN_V1ALPHA1", "SHIPWRIGHT_BUILDRUN", "SHIPWRIGHT_BUILD", "SHIPWRIGHT_BUILD_V1ALPHA1"]
     }
   },
   {
-    "type": "console.navigation/resource-ns",
+    "type": "console.navigation/href",
     "properties": {
+      "id": "shipwright-tablistpage-7",
+      "section": "builds",
       "perspective": "admin",
+      "prefixNamespaced": true,
+      "href": "/shipwright.io",
+      "name": "%shipwright-plugin~Shipwright%"
+    },
+    "flags": {
+      "required": ["SHIPWRIGHT_CLUSTERBUILDSTRATEGY_V1ALPHA1"],
+      "disallowed": ["SHIPWRIGHT_BUILDSTRATEGY", "SHIPWRIGHT_BUILDSTRATEGY_V1ALPHA1", "SHIPWRIGHT_BUILDRUN_V1ALPHA1", "SHIPWRIGHT_BUILDRUN", "SHIPWRIGHT_BUILD", "SHIPWRIGHT_BUILD_V1ALPHA1"]
+    }
+  },
+  {
+    "type": "console.navigation/href",
+    "properties": {
+      "id": "shipwright-tablistpage-8",
       "section": "builds",
-      "id": "shipwright-buildruns",
-      "insertAfter": "shipwright-buildruns",
-      "name": "%shipwright-plugin~Shipwright BuildRuns%",
-      "model": {
-        "group": "shipwright.io",
-        "version": "v1beta1",
-        "kind": "BuildRun"
-      }
+      "perspective": "admin",
+      "prefixNamespaced": true,
+      "href": "/shipwright.io",
+      "name": "%shipwright-plugin~Shipwright%"
     },
     "flags": {
-      "required": ["SHIPWRIGHT_BUILDRUN"]
+      "required": ["SHIPWRIGHT_CLUSTERBUILDSTRATEGY"],
+      "disallowed": ["SHIPWRIGHT_CLUSTERBUILDSTRATEGY_V1ALPHA1", "SHIPWRIGHT_BUILDSTRATEGY", "SHIPWRIGHT_BUILDSTRATEGY_V1ALPHA1", "SHIPWRIGHT_BUILDRUN_V1ALPHA1", "SHIPWRIGHT_BUILDRUN", "SHIPWRIGHT_BUILD", "SHIPWRIGHT_BUILD_V1ALPHA1"]
+    }
+  },
```

Comment:
No need to have 8 entries for the navigation item. We just need 2 entires and required flag should be for `ShipwrightBuild` model.

```{
    "type": "console.flag/model",
    "properties": {
      "model": {
        "group": "operator.shipwright.io",
        "version": "v1alpha1",
        "kind": "ShipwrightBuild"
      },
      "flag": "SHIPWRIGHTBUILD_V1ALPHA1"
    }
  },
{
    "type": "console.flag/model",
    "properties": {
      "model": {
        "group": "operator.shipwright.io",
        "version": "v1beta1",
        "kind": "ShipwrightBuild"
      },
      "flag": "SHIPWRIGHTBUILD"
    }
  },
  {
    "type": "console.navigation/href",
    "properties": {
      "id": "shipwright-tablistpage",
      "section": "builds",
      "perspective": "admin",
      "prefixNamespaced": true,
      "href": "/shipwright.io",
      "name": "%shipwright-plugin~Shipwright%"
    },
    "flags": {
      "required": ["SHIPWRIGHTBUILD"]
      "disallowed": ["SHIPWRIGHTBUILD_V1ALPHA1"]
    }
  },
{
    "type": "console.navigation/href",
    "properties": {
      "id": "shipwright-tablistpage",
      "section": "builds",
      "perspective": "admin",
      "prefixNamespaced": true,
      "href": "/shipwright.io",
      "name": "%shipwright-plugin~Shipwright%"
    },
    "flags": {
      "required": ["SHIPWRIGHTBUILD_V1ALPHA1"],
       "disallowed": ["SHIPWRIGHTBUILD"]
    }
  }```

Reply:
once `ShipwrightBuild` CR is available in the cluster then we can expect that other resources like buildRun and buildStrategy are also present.